etc/ATSCam/primary_hdu.header

```diff
@@ -10,14 +10,20 @@ MJD-OBS =                      / Modified Julian Date of observation
 OBS-LONG=           -70.749417 / [deg] Observatory east longitude
 OBS-LAT =           -30.244639 / [deg] Observatory latitude
 OBS-ELEV=               2663.0 / [m] Observatory elevation
+OBS-LONG=           -70.749417 / [deg] Observatory east longitude
+OBS-LAT =           -30.244639 / [deg] Observatory latitude
+OBS-ELEV=               2663.0 / [m] Observatory elevation
+OBSGEO-X=           1818938.94 / [m] X-axis Geocentric coordinate
+OBSGEO-Y=          -5208470.95 / [m] Y-axis Geocentric coordinate
+OBSGEO-Z=          -3195172.08 / [m] Z-axis Geocentric coordinate
 OBS-NITE=                      / Night of the observation
 TELESCOP= 'LSST AuxTelescope'  / Telescope name
 INSTRUME= 'LATISS'             / Instrument used to obtain these data
 RATEL   =                      / Telescope RA of observation (hr)
 DECTEL  =                      / Telescope DEC of observation (hr)
 RA      =                      / RA of Target (hr)
 DEC     =                      / DEC of Target (deg)
-ROTANGLE=                      / Rotation angle (deg)
+ROTANGLE=                      / Rotation angle relative to the sky (deg)
```

Comment:
Yes, we could add `ROTCOORD` to the header templates, but we'll also need to identify where this will be coming. I could add to the templates for now, but it won't be populated until the proper messages/event is present in SAL. Let me know if this is not OK.

Reply:
I think if you know that the camera is always publishing it in the same coordinates then it's still better to have the separate keyword. If the camera system can change then we insist that this becomes something that can be read from the system.

Reply:
Unfortunately, nothing has been defined by camera or by telescope and site yet. The camera won't publish any coordinates as far as I know. These would only come for T&S. I created a few tickets to make these request a while back.